src/api/attendanceApi.ts

```diff
@@ -36,4 +46,29 @@ const useGetAttendanceInfoListQuery = ({ memberId, year }: { memberId: number; y
   return useQuery<CallenderChartInfo[]>(attendanceKeys.attendanceInfoList({ memberId, year }), fetcher);
 };
 
-export { useGetTodayAttendancePointQuery, useGetTodayAttendanceInfoQuery, useGetAttendanceInfoListQuery };
+const useGetTodayAttendanceRank = ({ page, size = 10 }: PageAndSize) => {
+  const fetcher = () =>
+    axios
+      .get('/attendances/today-rank', {
+        params: { page, size },
+      })
+      .then(({ data }) => data);
+
+  return useQuery<TodayAttendRank>(attendanceKeys.todayAttendanceRank({ page, size }), fetcher, {
+    keepPreviousData: true,
+  });
+};
```

Comment:
저희 params 변수 따로 빼기로 한 거 요렇게 어떤가요?

```suggestion
const useGetTodayAttendanceRank = ({ page, size = 10 }: PageAndSize) => {
  const params = { page, size };

  const fetcher = () =>
    axios
      .get('/attendances/today-rank', {
        params,
      })
      .then(({ data }) => data);

  return useQuery<TodayAttendRank>(attendanceKeys.todayAttendanceRank(params), fetcher, {
    keepPreviousData: true,
  });
};
```

Reply:
반영했습니다!